docs/guides/admin/docs/configuration/basic.md

```diff
@@ -1,108 +1,114 @@
 Basic Configuration
 ===================
 
-<!-- _Guide says nothing about how to start running OC, nor does it point to the "Next Step"
-as the Installation Docs do (i.e. Install doc refers to Config doc) -->
-This guide will help you to change the basic configuration settings which are required or at least strongly recommended
-for each Opencast installation. This is basically what you should do, right after installing Opencast on your machine.
+This guide will help you to adjust the basic configuration settings strongly recommended for each Opencast installation.
```

Comment:
I would read `this guide` as referring to this specific page, rather than the OC docs as a whole.  Perhaps `This page` instead of `This guide` would clarify?

Reply:
As far as I can tell, the docs use `This guide`, `This page`, and `This document` pretty interchangeably. So I suppose it can stay as is.

Reply:
Just explicitly named this page:

```diff
diff --git a/docs/guides/admin/docs/configuration/basic.md b/docs/guides/admin/docs/configuration/basic.md
index aa5fcf8dcf..d7821eecaf 100644
--- a/docs/guides/admin/docs/configuration/basic.md
+++ b/docs/guides/admin/docs/configuration/basic.md
@@ -1,7 +1,7 @@
 Basic Configuration
 ===================
 
-This guide will help you to adjust the basic configuration settings strongly recommended for each Opencast installation.
+The basic configuration guide will help you to adjust the settings strongly recommended for each Opencast installation.
 This is what you should do right after installing Opencast.
 There are alternatives for some of these settings but this is the recommended setup.
 

```